pkgs/by-name/ch/changelog-d/package.nix

```diff
@@ -13,17 +13,11 @@ let
   haskellModifications =
     lib.flip lib.pipe [
       addCompletions
-      haskell.lib.doJailbreak
       haskell.lib.justStaticExecutables
     ];
 
   mkDerivationOverrides = finalAttrs: oldAttrs: {
 
-    version = oldAttrs.version + "-git-${lib.strings.substring 0 7 oldAttrs.src.rev}";
-
-    # nix-shell ./maintainers/scripts/update.nix --argstr package changelog-d
-    passthru.updateScript = lib.getExe (callPackage ./updateScript.nix { });
```

Comment:
Using a stable version breaks ./updateScript.nix. On the other hand, nix-update should now work. It won't update the list of dependencies, but those shouldn't change much.

Since the haskellPackages part of this packages will no longer be overwritten it might make sense to merge the two files. Should I do that?